src/data/announcementData.js

```diff
@@ -0,0 +1,10 @@
+import { Card } from '../../tests/e2e/fixtures/cards';
+
+export const announcementData = {
+  id: 'valueToLookForInLocalStorage',
+  activatorText: 'What shows up in the banner',
+  title: 'Heading',
+  imgSrc: null,
+  displayCards: [Card.ACE_OF_SPADES, Card.FIVE_OF_CLUBS, Card.TWO_OF_CLUBS, Card.EIGHT_OF_HEARTS],
```

Comment:
I like keeping this here, unless we need specific cards, I think it's a nice little detail